Yoast/wordpress-seo
# Help center styling tweaks

## Summary

This PR can be summarized in the following changelog entry:

* Adjusted Help Center styling for integration in WordPress

## Relevant technical choices:

- adjusts the left-right alignments of the Yoast containers (the "papers")
- increases a bit the bottom margin of the Need Help? button
- makes the KB heading and search field look like they should
- adds focus style to the KB search field
- makes the Get support heading look like it should
- makes the Get support "premium advantages" list look like a list
- introduces a CSS class `yoast-measure` to apply the better readability `max-width: 600px`
- removes redundant wrapper with extraneous `classID` prop around the Help Center
- minor: simplifies the debug info markup and use specific styling to avoid reusing WP CSS rules

## Test instructions

This PR can be tested by following these steps:

* check both the settings pages and the post metabox to verify the Help Center looks as intended

Fixes https://github.com/Yoast/yoast-components/issues/329

Screenshots:

KB heading and search field before:

<img width="895" alt="screen shot 2017-10-09 at 11 44 43" src="https://user-images.githubusercontent.com/1682452/31332605-50b81ae6-ace7-11e7-9e3c-8630264e97c4.png">


KB heading and search field after (with focus style):

<img width="884" alt="screen shot 2017-10-09 at 11 41 45" src="https://user-images.githubusercontent.com/1682452/31332539-101d5e88-ace7-11e7-92d1-b18bc52cd98a.png">

<img width="886" alt="screen shot 2017-10-09 at 11 41 51" src="https://user-images.githubusercontent.com/1682452/31332540-102b324c-ace7-11e7-81de-a769a07af965.png">

Get Support tab before and after:

<img width="890" alt="screen shot 2017-10-09 at 11 42 12" src="https://user-images.githubusercontent.com/1682452/31332557-1db39382-ace7-11e7-945a-049f8e5392d5.png">
<img width="901" alt="screen shot 2017-10-09 at 11 42 48" src="https://user-images.githubusercontent.com/1682452/31332556-1db1d934-ace7-11e7-9deb-49e0261c5f42.png">
